### ISSUE TITLE
Could you please share your solver?

### ISSUE THREAD

**Issue by leon-liangwu** [miaow1988/ShuffleNet_V2_pytorch_caffe#6]
I can not train the shufflenet to the accuracy in the paper, and there is a large margin between mine and that in the paper.
Could you please share your solver.prototxt?

**Comment by miaow1988** [comment]
I trained my models with PyTorch. The Caffe model files are converted from my PyTorch models.
Just use follow parameters should be fine.
1. Iter: 300000
2. Batch Size = 1024
3. Optimizer: SGD
4. LR: from 0.5, linearly decay
5. WD: 0.00004

**Comment by ArsenLuca** [comment]
@miaow1988 so what's your linearly-decay ratio?

**Comment by miaow1988** [comment]
Just use the linear decay, as same as in the original paper.
